app/views/recipes/show.html.erb

```diff
@@ -31,7 +31,11 @@
     <p><%= @recipe.description %></p>
     <br>
 
-    <%= button_to "Add ingredient", new_user_recipe_recipefood_path(current_user, @recipe), method: :get %> <br>
+    <div class="recipe-show-btns">
+      <button> Generate Shopping List </button>
```

Comment:
- [ Required ] Good job adding the `Generate shopping list` button, however, you still need to ensure that the button leads to the `General shopping list` page as specified in the project requirements.

![](https://github.com/microverseinc/curriculum-rails/raw/main/recipe-app/images/general_shopping_list.png)

## Project requirements
![Screenshot from 2023-04-20 21-36-22](https://user-images.githubusercontent.com/16392046/233457724-174015cf-c163-462e-8e83-d387a54c3a4a.png)